api/migrations/20210929000000-alter-device-columns.js

```diff
@@ -30,11 +30,13 @@ module.exports = {
       queryInterface.removeColumn("Devices", "lastRecordingTime"),
       queryInterface.removeColumn("Groups", "lastRecordingTime"),
       queryInterface.removeColumn("Recordings", "fileSize", Sequelize.INTEGER),
-      queryInterface.removeColumn("Recordings", "rawFileSize", Sequelize.INTEGER),
+      queryInterface.removeColumn(
```

Comment:
Wouldn't you need to start a new migration file for these changes, since these old migrations have already been applied?